### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by ritazakhodyaeva**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,35 +1,45 @@
...
-4. Place the fields within the Pivot Grid Control areas ([Data Header Area](https://docs.devexpress.com/WindowsForms/1688/controls-and-libraries/pivot-grid/ui-elements/data-header-area), [Column Header Area](https://docs.devexpress.com/WindowsForms/1686/controls-and-libraries/pivot-grid/ui-elements/column-header-area), [Row Header Area](https://docs.devexpress.com/WindowsForms/1685/controls-and-libraries/pivot-grid/ui-elements/row-header-area) or [Filter Header](https://docs.devexpress.com/WindowsForms/1684/controls-and-libraries/pivot-grid/ui-elements/filter-header-area)) as required and make them visible.
+3. Create Pivot Grid fields for OLAP measures and dimensions in following ways:
```
Comment (albertov05) [inline Readme.md]: Can you specify whether the list items below are _alternatives_?
For example, do you mean "in one of the following ways"?
Or are they steps?

File: Readme.md [hunk 2]
```diff
@@ -1,35 +1,45 @@
...
-4. Place the fields within the Pivot Grid Control areas ([Data Header Area](https://docs.devexpress.com/WindowsForms/1688/controls-and-libraries/pivot-grid/ui-elements/data-header-area), [Column Header Area](https://docs.devexpress.com/WindowsForms/1686/controls-and-libraries/pivot-grid/ui-elements/column-header-area), [Row Header Area](https://docs.devexpress.com/WindowsForms/1685/controls-and-libraries/pivot-grid/ui-elements/row-header-area) or [Filter Header](https://docs.devexpress.com/WindowsForms/1684/controls-and-libraries/pivot-grid/ui-elements/filter-header-area)) as required and make them visible.
+3. Create Pivot Grid fields for OLAP measures and dimensions in following ways: 
 
-Use the invoked [Customization Form](https://docs.devexpress.com/WindowsForms/1805) to arrange fields.
+- Use the [PivotGridControl.RetrieveFields](https://docs.devexpress.com/WindowsForms/DevExpress.XtraPivotGrid.PivotGridControl.RetrieveFields(DevExpress.XtraPivotGrid.PivotArea-System.Boolean)) method that create fields, moves them to the specified area, and makes them hidden.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
- Use the [PivotGridControl.RetrieveFields](https://docs.devexpress.com/WindowsForms/DevExpress.XtraPivotGrid.PivotGridControl.RetrieveFields(DevExpress.XtraPivotGrid.PivotArea-System.Boolean)) method, which creates fields, moves them to the specified area, and hides them.
```

File: Readme.md [hunk 3]
```diff
@@ -1,35 +1,45 @@
...
+- Use [DataSourceColumnBinding](https://docs.devexpress.com/WindowsForms/DevExpress.XtraPivotGrid.DataSourceColumnBinding)) and [OLAPExpressionBinding](https://docs.devexpress.com/WindowsForms/DevExpress.XtraPivotGrid.OLAPExpressionBinding) to bind Pivot Grid fields to data.
 
+4. Specify field settings. Use the field's [Area](https://docs.devexpress.com/CoreLibraries/DevExpress.XtraPivotGrid.PivotGridFieldBase.Area) property to place the fields within the Pivot Grid Control areas: [Data Header Area](https://docs.devexpress.com/WindowsForms/1688/controls-and-libraries/pivot-grid/ui-elements/data-header-area), [Column Header Area](https://docs.devexpress.com/WindowsForms/1686/controls-and-libraries/pivot-grid/ui-elements/column-header-area), [Row Header Area](https://docs.devexpress.com/WindowsForms/1685/controls-and-libraries/pivot-grid/ui-elements/row-header-area) or [Filter Header](https://docs.devexpress.com/WindowsForms/1684/controls-and-libraries/pivot-grid/ui-elements/filter-header-area). Set the field's [Visible](https://docs.devexpress.com/CoreLibraries/DevExpress.XtraPivotGrid.PivotGridFieldBase.Visible) property to `visible`.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
4. Specify field settings. Use the field's [Area](https://docs.devexpress.com/CoreLibraries/DevExpress.XtraPivotGrid.PivotGridFieldBase.Area) property to place the fields within the Pivot Grid Control areas: [Data Header Area](https://docs.devexpress.com/WindowsForms/1688/controls-and-libraries/pivot-grid/ui-elements/data-header-area), [Column Header Area](https://docs.devexpress.com/WindowsForms/1686/controls-and-libraries/pivot-grid/ui-elements/column-header-area), [Row Header Area](https://docs.devexpress.com/WindowsForms/1685/controls-and-libraries/pivot-grid/ui-elements/row-header-area), or [Filter Header Area](https://docs.devexpress.com/WindowsForms/1684/controls-and-libraries/pivot-grid/ui-elements/filter-header-area). Set the field's [Visible](https://docs.devexpress.com/CoreLibraries/DevExpress.XtraPivotGrid.PivotGridFieldBase.Visible) property to `visible`.
```

File: Readme.md (hunk 1 above)
Comment (ritazakhodyaeva) [reply Readme.md]:
```suggestion
3. Create Pivot Grid fields for OLAP measures and dimensions in one of the following ways: 

```

**Pull request merged**